src/ir/typescript_types.py

```diff
@@ -237,3 +322,8 @@ def __init__(self, nr_type_parameters: int):
         self.nr_type_parameters = nr_type_parameters
         super().__init__(name, type_parameters)
         self.supertypes.append(ObjectType())
+
+
+# Literal Types
+
+literal_types = LiteralTypes(10, 10)
```

Comment:
add constant variables to avoid magic numbers, e.g.,

```python
MAX_NUM_LITERAL_TYPES = 10
MAX_STRING_LITERAL_TYPES = 10
literal_types = LiteralTypes(MAX_STRING_LITERAL_TYPES, MAX_NUM_LITERAL_TYPES)
```

Also, add a `TODO` comment that the above limits should become user-configurable.